pkg/ha/ha_tracker.go

```diff
@@ -212,6 +218,12 @@ func NewHATracker(cfg HATrackerConfig, limits HATrackerLimits, trackerStatusConf
 		}
 		t.client = client
 	}
+	go func() {
+		for {
+			time.Sleep(10 * time.Second)
+			t.updateUserReplicaGroupCount()
+		}
+	}()
```

Comment:
Can we move this to the `loop` method?
You can follow `cleanupOldReplicasLoop` and  do it similarly.

Reply:
okay :)